charts/cactus-backend/templates/workload.yaml

```
@@ -111,7 +115,24 @@ spec:
       topologySpreadConstraints:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-  {{- if .Values.persistentVolume.enabled }}
+  {{- if or (not (and (eq .Values.kind "StatefulSet") .Values.persistentVolume.enabled)) .Values.configFiles.enabled }}
+      volumes:
+      {{- if not (and (eq .Values.kind "StatefulSet") .Values.persistentVolume.enabled) }}
+        - name: storage-volume
+          emptyDir:
+          {{- if .Values.emptyDir.sizeLimit }}
+            sizeLimit: {{ .Values.emptyDir.sizeLimit }}
+          {{- else }}
+            sizeLimit: 8Gi
+          {{- end }}
```

I think it can be simplified like this:
```suggestion
            sizeLimit: {{ .Values.emptyDir.sizeLimit | default "8Gi" }}
```
But actually since we're in control of the `values.yaml` and already set the default `8Gi` there, we can almost assume that nobody will actually override and set it to `""` or some other invalid value and still expect it to work.